app/events/views.py

```diff
@@ -0,0 +1,378 @@
+"""import depancies and methods."""
+
+from flask.views import MethodView
+from flask import make_response, request, jsonify
+from app.models import User, Events
+
+from . import events_blueprint
+
+
```

Comment:
Great work...I like your workflow. The abstractions make your code nice to read.

Reply:
Thank you for the review

Reply:
okay I now understand your review